lib/elements/dom-repeat.js

```diff
@@ -347,11 +347,11 @@ export class DomRepeat extends domRepeatBase {
           /** @type {!HTMLElement} */ (this));
       let template = this.template = thisAsTemplate._templateInfo ?
           thisAsTemplate :
-          /** @type {!HTMLTemplateElement} */ (this.querySelector('template'));
+          /** @type {!HTMLTemplateElement} */ (wrap(this).querySelector('template'));
```

Comment:
nbd, but nothing should ever be in a `dom-repeat` or `dom-if` except for a template (i.e. no nested elements with SR's that would need to be filtered out), so this is basically fine to always be native qs

Reply:
Dome